bower.json

```diff
@@ -16,21 +16,22 @@
   ],
   "dependencies": {
     "purescript-bifunctors": "^4.0.0",
-    "purescript-control": "^4.0.0",
```

Comment:
Please correct me if I'm wrong but it looks to me like all of these changes are just tightening lower bounds unnecessarily. If so, we shouldn't do that for core libraries. I think the only change in this file should be the addition of the minibench dependency.